meta-balena-common/recipes-support/os-helpers/os-helpers/os-helpers-bootloader-config

```diff
@@ -0,0 +1,110 @@
+if [ -f "/usr/libexec/os-helpers-logging" ]; then
```

Comment:
Hey @acostach, I'm wondering who is going to call this in the end
 Do we want to filter on allowed modification even for non SB devices?